plugins/woocommerce/src/StoreApi/Routes/V1/Agentic/CheckoutSessions.php

```diff
@@ -0,0 +1,452 @@
+<?php
+declare(strict_types=1);
+namespace Automattic\WooCommerce\StoreApi\Routes\V1\Agentic;
+
+use Automattic\WooCommerce\StoreApi\Routes\V1\AbstractCartRoute;
+use Automattic\WooCommerce\StoreApi\SchemaController;
+use Automattic\WooCommerce\StoreApi\Schemas\V1\AbstractSchema;
+use Automattic\WooCommerce\StoreApi\Utilities\CartController;
+use Automattic\WooCommerce\StoreApi\Utilities\OrderController;
+use Automattic\WooCommerce\Internal\Features\FeaturesController;
+
+/**
+ * CheckoutSessions class.
+ *
+ * Handles the Agentic Checkout API checkout sessions endpoint.
+ * This endpoint allows AI agents to create and manage checkout sessions.
+ */
+class CheckoutSessions extends AbstractCartRoute {
+	/**
+	 * The route identifier.
+	 *
+	 * @var string
+	 */
+	const IDENTIFIER = 'agentic-checkout-sessions';
+
+	/**
+	 * The route's schema type.
+	 *
+	 * @var string
+	 */
+	const SCHEMA_TYPE = 'agentic-checkout-session';
+
+	/**
+	 * Order controller for managing draft orders.
+	 *
+	 * @var OrderController
+	 */
+	protected $order_controller;
+
+	/**
+	 * Cart controller for managing cart operations.
+	 *
+	 * @var CartController
+	 */
+	protected $cart_controller;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param SchemaController $schema_controller Schema Controller instance.
+	 * @param AbstractSchema   $schema Schema class instance.
+	 */
+	public function __construct( $schema_controller, $schema ) {
+		parent::__construct( $schema_controller, $schema );
+		$this->order_controller = new OrderController();
+		$this->cart_controller  = new CartController();
+	}
+
+	/**
+	 * Get the path of this REST route.
+	 *
+	 * @return string
+	 */
+	public function get_path() {
+		return '/checkout_sessions';
```

Comment:
Should this just be:

```suggestion
		return self::get_path_regex();
```